### PR TITLE
Add a delay to study bursts.

### PR DESCRIPTION
**Pull request by alxdarksage**

### CODE REVIEW

File: src/test/java/org/sagebionetworks/bridge/services/StudyActivityEventServiceTest.java [hunk 1]
```diff
@@ -431,6 +432,61 @@ public void publishEvent_publishesStudyBursts() {
...
+        
+        service.publishEvent(event, false, true);
+        
+        verify(mockDao, times(4)).publishEvent(eventCaptor.capture());
+        
+        StudyActivityEvent origin = eventCaptor.getAllValues().get(0);
+        assertEquals(origin.getEventId(), "enrollment");
+        
+        StudyActivityEvent sb1 = eventCaptor.getAllValues().get(1);
+        assertEquals(sb1.getEventId(), "study_burst:foo:01");
```
Comment (alxdarksage) [inline src/test/java/org/sagebionetworks/bridge/services/StudyActivityEventServiceTest.java:468]: All of this could be removed

Comment (alxdarksage) [reply src/test/java/org/sagebionetworks/bridge/services/StudyActivityEventServiceTest.java:468]: removed

File: src/test/java/org/sagebionetworks/bridge/validators/Schedule2ValidatorTest.java [hunk 2]
```diff
@@ -214,6 +214,14 @@ public void studyBurstOriginEventIdCannotBeBlank() {
         
         assertValidatorMessage(INSTANCE, schedule, STUDY_BURSTS_FIELD + "[0]." + ORIGIN_EVENT_ID_FIELD, CANNOT_BE_BLANK);
     }
+    
+    @Test
+    public void studyBurstDelayCannotBeStrangPeriod() {
```
Comment (jnelsonjava) [inline src/test/java/org/sagebionetworks/bridge/validators/Schedule2ValidatorTest.java]: typo in word: strange

File: src/test/java/org/sagebionetworks/bridge/validators/Schedule2ValidatorTest.java [hunk 3]
```diff
@@ -223,6 +231,14 @@ public void studyBurstIntervalCannotBeNull() {
         assertValidatorMessage(INSTANCE, schedule, STUDY_BURSTS_FIELD + "[0]." + INTERVAL_FIELD, CANNOT_BE_NULL);
     }
 
+    @Test
+    public void studyBurstIntervalCannotBeStrangPeriod() {
```
Comment (jnelsonjava) [inline src/test/java/org/sagebionetworks/bridge/validators/Schedule2ValidatorTest.java]: same as above

**Review by jnelsonjava — APPROVED** [review]
Looks good. There is a spelling edit to consider but no functional notes.

File: src/test/java/org/sagebionetworks/bridge/validators/Schedule2ValidatorTest.java (hunk 3 above)
Comment (alxdarksage) [reply src/test/java/org/sagebionetworks/bridge/validators/Schedule2ValidatorTest.java]: Fixed, also why did I use the word "strange" and not "invalid" — fixed

**Pull request merged**